### PR TITLE
login (landing page) and create new user page

### PR DESCRIPTION
**Pull request by MacyMcAnally**
Please review this login branch for Login (landing page) and create new user page!

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...7ProjectRepo\\src\\reportWebVitals.js":"2","C:\\Users\\dexlu\\OneDrive\\Desktop\\Training\\Project2\\Team7ProjectRepo\\src\\App.js":"3"},{"size":500,"mtime":499162500000,"results":"4","hashOfConfig":"5"},{"size":362,"mtime":499162500000,"results":"6","hashOfConfig":"5"},{"size":528,"mtime":499162500000,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"nop1jn",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\dexlu\\OneDrive\\Desktop\\Training\\Project2\\Team7ProjectRepo\\src\\index.js",[],["15","16"],"C:\\Users\\dexlu\\OneDrive\\Desktop\\Training\\Project2\\Team7ProjectRepo\\src\\reportWebVitals.js",[],"C:\\Users\\dexlu\\OneDrive\\Desktop\\Training\\Project2\\Team7ProjectRepo\\src\\App.js",[],{"ruleId":"17","replacedBy":"18"},{"ruleId":"19","replacedBy":"20"},"no-native-reassign",["21"],"no-negated-in-lhs",["22"],"no-global-assign","no-unsafe-negation"]
```
Comment (tylercodes1) [inline .eslintcache]: add .eslintcache to the .gitignore

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,32 @@
...
+  margin-left: 250px;
+  background-color: rgba(0, 0, 0, 0.4);
+  transition: 400ms;
+  /* transition: var(--transition-delay); */
 }
+.showcase.shank {
+  margin-left: 0;
+  background-color: white;
+  transition: var(--transition-delay);
+}
```
Comment (tylercodes1) [inline src/App.css]: Please revert css of App.css back to it's original styling (red)

File: src/pages/LoginPage.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './FormStyle.css';
+
+function LoginPage() {
+    return (
+        <div className="login">
```
Comment (tylercodes1) [inline src/pages/LoginPage.js]: please rename the class to "login-page" and change the according css mapping

File: src/pages/CreateNewUserPage.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './FormStyle.css';
+
+function CreateNewUserPage() {
+    return (
+        <div className="newuser">
```
Comment (tylercodes1) [inline src/pages/CreateNewUserPage.js]: change className to "newuser-page" and change the according css mapping

File: src/pages/FormStyle.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
+.login {
+    position: relative;
+    width: 400px;
+    height: 350px;
+    padding: 30px 38px;
+    margin-top: 150px;
+    margin-left: 600px;
```
Comment (tylercodes1) [inline src/pages/FormStyle.css]: No changes here, but we will change how this is rendered on the screen to be "responsive-design"

File: src/pages/CreateNewUserPage.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (tylercodes1) [inline src/pages/CreateNewUserPage.js]: Please change the file structure as follows
>src
>>pages
>>>LoginPage
>>>>LoginPage.js
>>>>LoginPage.css

>>>CreateNewUserPage
>>>>CreateNewUserPage.js
>>>>CreateNewUserPage.css

**Review by tylercodes1 — CHANGES_REQUESTED** [review]
Styling is pretty good, may not need to change it later. Please review the comments for structural changes

File: src/pages/LoginPage.js (hunk 3 above)
Comment (MacyMcAnally) [reply src/pages/LoginPage.js:6]: Do you mean the classname in the div tag?

**Comment by MacyMcAnally** [comment]
Please review! Thank you!

**Review by tylercodes1 — APPROVED** [review]
Nice work!

**Comment by MacyMcAnally** [comment]
Hey @tylercodes1, I brought in the latest changes in main and cleaned up my history with rebase. Is everything ok to merge still?

**Pull request merged**